pkg/disttask/importinto/planner.go

```diff
@@ -482,7 +482,7 @@ func getSortedKVMetasForIngest(planCtx planner.PlanCtx) (map[string]*external.So
 	for kvGroup, kvMeta := range kvMetasOfEncodeStep {
 		// only part of kv files are merge sorted. we need to merge kv metas that
 		// are not merged into the kvMetasOfMergeSort.
-		if skipMergeSort(kvGroup, kvMeta.MultipleFilesStats) {
+		if !p.Plan.ForceMergeStep && skipMergeSort(kvGroup, kvMeta.MultipleFilesStats) {
```

Comment:
maybe we can remove the failpoint, `ForceMergeStep` seems cover all calling

Reply:
i deleted it once, but find that some test only want to skip for some kv group